src/Shrubbery/Classes.hs

```diff
@@ -50,14 +50,14 @@ import Shrubbery.TypeList (FirstIndexOf, KnownLength)
 
 {-|
   This type family is used by both 'Dissection' and 'Unification' to specify
-  the types of the values available in the branching type. IF you provided
```

Comment:
this one might have been merely for extra emphasis ;)

Reply:
:laughing: It was not